Short-Answer/Algorithms_Answers.md

```diff
@@ -2,13 +2,13 @@
 
 ## Exercise I
 
-a)
+a) O(n), this question only requires one loop
 
 
-b)
+b) O(n^2), this question has a nested loop as well. two loops total
 
 
-c)
+c) O(log n), recursive
```

Comment:
Answer: O(n)
"How many times will we recursively call bunnyEars() as bunnies gets bigger?" Look at how the parameters change in the recursive call. Since we are only decrementing bunnies by 1 every time, the function will be called n times. Equivalent to a loop from 0 to bunnies. We call recursively with bunnies-1each time.